File: Week 9 - Full Stack/RestaurantFavesFrontend/RestaurantFrontend/src/app/order.ts
// To parse this data:
//
//   import { Convert, Order } from "./file";
//
//   const order = Convert.toOrder(json);

//This model needs to match the spelling and casing of the C# API 
//The C# model needs to match the SQL database
export interface Order {
    id:          number;
    description: string;
    restaurant:  string;
    rating:      number;
    orderAgain:  boolean;
}

// Converts JSON strings to/from your types
export class Convert {
    public static toOrder(json: string): Order {
        return JSON.parse(json);
    }

    public static orderToJson(value: Order): string {
        return JSON.stringify(value);
    }
}

File: Week 9 - Full Stack/RestuarantFaves/RestuarantFaves/Models/Order.cs
using System.ComponentModel.DataAnnotations;

namespace RestuarantFaves.Models
{
    public class Order
    {
        [Key]
        public int id { get; set; }

        [MaxLength(80)]
        public string Description { get; set; }

        [MaxLength(30)]
        public string Restaurant { get; set; }

        [Range(1,5)]
        public int Rating { get; set; }

        public bool OrderAgain { get; set; }
    }
}
